nav2_controller/src/nav2_controller.cpp

```diff
@@ -321,6 +322,13 @@ void ControllerServer::computeAndPublishVelocity()
     pose,
     nav_2d_utils::twist2Dto3D(twist));
 
+  std::shared_ptr<Action::Feedback> feedback = std::make_shared<Action::Feedback>();
+  feedback->speed = cmd_vel_2d.twist.linear.x;
```

Comment:
```suggestion
  feedback->speed = std::hypot(cmd_vel_2d.twist.linear.x, cmd_vel_2d.twist.linear.y);
```
To support holonomic robots.